draft-ietf-trans-gossip.md

```diff
@@ -515,12 +536,18 @@ An HTTPS client will retrieve SCTs from an HTTPS server, and must
 obtain an inclusion proof to an STH in order to verify the promise
 made by the SCT.
 
-An HTTPS client may receive SCT bundled with an inclusion proof to a
+An HTTPS client may also receive SCT bundled with an inclusion proof to a
 historical STH via an unspecified future mechanism. Because this
 historical STH is considered personally identifiable information per
 above, the client must obtain a consistency proof to a more recent
 STH.
 
+A client SHOULD perform proof fetching. A client MUST NOT perform 
+proof fetching for any SCTs or STHs issued by a locally added log. 
+A client MAY fetch an inclusion proof for an SCT (issued by a 
+pre-loaded log) that validates a certificate chaining to a locally 
+added trust anchor.
+
```

Comment:
It'd give me the proof I wanted. :) Maybe the issue here is that a gossip draft forbids a client to do something useful outside the scope of gossip.